pulser/waveforms.py

```diff
@@ -739,6 +739,94 @@ def __mul__(self, other: float) -> InterpolatedWaveform:
         )
 
 
+class KaiserWaveform(Waveform):
+    """A Kaiser window of a specified duration and beta parameter.
+
```

Comment:
Can you write a bit about this window? Maybe a `See Also` for the numpy docs? I feel like it's necessary to explain the role of the `beta` parameter, somehow.